index.html

```diff
@@ -5164,6 +5164,116 @@ <h2 id="AudioWorklet-concepts">
           </dd>
         </dl>
         <section>
+          <h2 id="AudioWorkletGlobalScope">
+            The <dfn>AudioWorkletGlobalScope</dfn> interface
+          </h2>
+          <p>
+            This special execution context is designed to enable the
+            generation, processing, and analysis of audio data directly using
+            JavaScript in the audio <a>rendering thread</a>. The user-supplied
+            script code is evaluated in this scope to define an
+            <a>AudioWorkletProcessor</a>.
+          </p>
+          <p>
+            <a>AudioWorkletGlobalScope</a> has a <dfn>node name to processor
```

Comment:
I expanded it. However, `dfn` here does not work well. It still complains about a linkless `a` on `node name to processor definition map`.